packages/core/strapi/ee/index.js

```diff
@@ -165,6 +165,10 @@ module.exports = Object.freeze({
     return ee.enabled;
   },
 
+  get licenseInfo() {
+    return ee.licenseInfo;
```

Comment:
This object is not frozen, thus it can too easily be modified outside of this file.
You can either return the number of seats directly or freeze it here or when assigned.